### PR TITLE
Resolve issue #221

### PR DESCRIPTION
**Pull request by mrtpain**
* Added link to open wallet directory in advanced -> about component
* Clean about component unused dependencies
* Refactor using async pipe in template rather than sub in ts file

### CODE REVIEW

File: StratisCore.UI/src/app/wallet/advanced/components/about/about.component.ts [hunk 1]
```diff
@@ -1,55 +1,31 @@
...
-      );
+    this.nodeStatusSubscription$ = this.apiService.getNodeStatusInterval();
   }
 
-  private cancelSubscriptions() {
-    if(this.nodeStatusSubscription) {
-      this.nodeStatusSubscription.unsubscribe();
-    }
+  openWalletDirectory(directory: string): void {
+    this.electron.shell.showItemInFolder(directory);
```
Comment (dev0tion) [inline StratisCore.UI/src/app/wallet/advanced/components/about/about.component.ts]: We also have a mode that allows Core to run in the browser, in that case you won't have access to the electron API.
Please alter the PR the include something like this: https://github.com/stratisproject/StratisCore/blob/master/StratisCore.UI/src/app/shared/services/global.service.ts#L115

**Comment by mrtpain** [comment]
if variables are needed for calcs in the component ts files you can use something like 
`myObservable$ = this.service.getSomething().pipe(.tap(response => this.var = response));`

This allows for async pipe use for reactive templates which also handles subs and unsubs. Then pipe and tap into subscription to run something in controller each time something is emitted. This would be a best practice scenario going forward but I understand your consistency concerns.

Moving forward with testing and code coverage, the async pipe eliminates a lot of unnecessary tests by letting angular handle the subscriptions where possible. The component ts file was reduced to half the amount of lines and this could be the case throughout several components.

File: StratisCore.UI/src/app/wallet/advanced/components/about/about.component.ts (hunk 1 above)
Comment (mrtpain) [reply StratisCore.UI/src/app/wallet/advanced/components/about/about.component.ts]: Good catch, will update.

**Comment by dev0tion** [comment]
Ok interesting approach, I like it.

File: StratisCore.UI/src/app/wallet/advanced/components/about/about.component.ts (hunk 1 above)
Comment (mrtpain) [reply StratisCore.UI/src/app/wallet/advanced/components/about/about.component.ts]: Added the check, no clickable feature if wallet is not served with electron

**Pull request merged**